packages/gnupg.rb

```diff
@@ -39,4 +39,9 @@ class Gnupg < Autotools
     --with-readline'
 
   run_tests
+
+  def self.patch
+    downloader 'https://files.gnupg.net/file/data/zy2tsxkydxq4okdhxj4p/PHID-FILE-cnig3h7b4hnrmr5efbh2/file', '7beaf4c72dc91a885fb088f7c1ef043ab49f230666727bcf2609954d722aba36', 'unistd.patch'
+    system 'patch -Np1 -i unistd.patch'
```

Comment:
Does `system "git apply unistd.patch"` work?

Reply:
I was just thinking this might be an easier way to do it.  Several other packages are using this method.

Reply:
```bash
grep -l "patch -Np1 -i" * | wc -l
55
grep -l "git apply" * | wc -l
14
```

I don't know if the `git apply` command is more prone to issues than using `patch -Np1 -i`.  I haven't done the research on that...